### DISCUSSION TITLE
Convert Postman scripts to Bruno

### DISCUSSION THREAD

**Discussion by vanloc1808** [usebruno/bruno#2276] [General]
Hello guys.
Are there any tutorials for writing Bruno scripts? I cannot find one.
I have this script in Postman, but don't know how to convert it to Bruno
```
pm.test("Has token", function () {
    var jsonData = pm.response.json();
    // pm.expect(jsonData).to.have.property("data");
    
    var token = jsonData.token;
    pm.expect(token).to.be.not.empty;
    pm.environment.set("token", token);
});
```

**Comment by blablapola** [comment]
i'm late but you can find in the bruno documentation the JS reference. It's near from PM scripting, just the method names change.
